db/seed_festivals_public.py

```
@@ -0,0 +1,364 @@
+        if code and name:
+            names[code] = name
+    return names
+
+
+def fetch_holidays(codes) -> dict:
+    """나라별 2026년 공휴일. Nager.Date 가 모르는 나라는 빈 목록으로 둔다"""
+    holidays = {}
+    for code in sorted(codes):
+        try:
+            holidays[code] = get_json(NAGER_URL.format(year=YEAR, code=code), timeout=30)
+        except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)